controllers/users.controller.js

```diff
@@ -33,12 +33,12 @@ async function sendEmail (data) {
       Il ne te reste plus qu'à valider ton adresse email en collant le lien ci-dessous dans ton navigateur :
       Toute l'équipe de Freelance Lyonnais te remercie.
 
-      "${process.env.EMAIL_DESTINATION_URL}/${data.email}_${data.key}"`,
+      "${process.env.EMAIL_DESTINATION_URL}${data.email}_${data.key}"`,
 
       html: `<p>Cher(e) Freelance Lyonnais,</Il>
       <p>Nous te remercions pour ton inscription sur notre site.</p>
       <p>Il ne te reste plus qu'à valider ton adresse email en copiant ou cliquant sur le lien ci-dessous :</p>
-      <a href=${process.env.EMAIL_DESTINATION_URL}/${data.email}/${data.key}>Vérification email</a>
+      <a href=${process.env.EMAIL_DESTINATION_URL}${data.email}/${data.key}>Vérification email</a>
```

Comment:
préférer une url plus "restful" de la forme /email_validation?email=<...>&key=<...>

Reply:
You mean in the describe / it comments ? We'll correct that issue in feature/validation branch